app/helpers/hosts_helper.rb

```diff
@@ -276,7 +276,7 @@ def overview_fields(host)
     fields += [[_("Operating System"), link_to(host.operatingsystem.to_label, hosts_path(:search => %{os_title = "#{host.operatingsystem.title}"}))]] if host.operatingsystem.present?
     fields += [[_("PXE Loader"), host.pxe_loader]] if host.operatingsystem.present? && !host.image_build?
     fields += [[_("Host group"), link_to(host.hostgroup, hosts_path(:search => %{hostgroup_title = "#{host.hostgroup}"}))]] if host.hostgroup.present?
-    fields += [[_("Uptime"), time_ago_in_words(host.uptime_seconds.seconds.from_now)]] if host.uptime_seconds.present?
+    fields += [[_("Boot time"), date_time_relative(host&.reported_data&.boot_time) || _('Not reported')]]
```

Comment:
Not reported is never going to show up as `date_time_relative` will handle nil value in the react component as 'N/A'.
Could you eighter remove the fallback, or handle it somewhat like (uglier code, but nicer value):
```suggestion
    fields += [[_("Boot time"), (boot_time = host&.reported_data&.boot_time) ? date_time_relative(boot_time) : _('Not reported')]]
```
Or add the option/quark `default_value` to the `date_time_relative` method and propagate it through `mount_date_component` to the react - the best solution :stuck_out_tongue:

Reply:
I took your suggestion, also squashed all back to just 2 commits since I feel we're getting closer, they should be merged separately at the end